scipy/stats/_hypotests.py

```diff
@@ -1433,3 +1435,242 @@ def cramervonmises_2samp(x, y, method='auto'):
             p = max(0, 1. - _cdf_cvm_inf(tn))
 
     return CramerVonMisesResult(statistic=t, pvalue=p)
+
+
+class TukeyKramerResult:
+    """Result of `scipy.stats.tukeykramer`.
+
+    Attributes
+    ----------
+    sig_level : float
+            The desired significance level (copied from `tukeykramer` input).
+    statistic : float ndarray
+        The computed statistic of the test for each comparison. The
+        ``(i, j)`` index is the p-value for the ith - jth group comparison.
```

Comment:
```suggestion
        The computed statistic of the test for each comparison. The element
        at index ``(i, j)`` is the statistic for the comparison between groups ``i``
        and ``j``.
```

Reply:
This comment feels left out : (

Reply:
This is still not fully addressed in `Tukey_HSDResult`. If you disagree with the comment, please let me know so we can address it that way.